pkgs/by-name/pu/pulumi-esc/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "pulumi-esc";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "pulumi";
+    repo = "esc";
+    rev = "v${version}";
+    hash = "sha256-fpS4bnS51Ch3dUvwqHdD4DBpZFZQCqwDlYWotyEfvlQ=";
+  };
+
+  subPackages = "cmd/esc";
+
+  vendorHash = "sha256-Wcll/UWgCTBd1Bkbdt6kZZ6jgouWkzkFicv3CvX5hw4=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=github.com/pulumi/esc/cmd/esc/cli/version.Version=${src.rev}"
```

Comment:
Brilliant, thank you for remembering to link in the version. Is src.rev going to be just "v0.9.0" no hash added?

Reply:
Yes, just comes out as "v0.9.0". Would you recommend doing it differently?

Reply:
That's great!